spec/unit/shoulda/matchers/active_record/validate_uniqueness_of_matcher_spec.rb

```diff
@@ -450,9 +450,19 @@
 
         expect(record).to validate_uniqueness
       end
+      context 'when the value has no case sensitivity' do
+        it 'accepts' do
+          model = define_model_validating_uniqueness(
+            attribute_type: :string,
+            validation_options: { case_sensitive: true },
+          )
+          record = create_record_from(model, attribute_name => '123')
```

Comment:
I thought that too, but changing that line to `record = create_record_from(model, value: '123')` makes the test fail.